test/test_exporter.py

```diff
@@ -571,6 +571,37 @@ def test_configuration_custom(self):
                 self.assertIsInstance(kwargs.get("exporter"),
                                       _DynatraceMetricsExporter)
 
+    @patch.object(requests.Session, 'post')
+    def test_nonstring_attributes_dropped(self, mock_post):
+        mock_post.return_value = self._get_session_response()
+
+        attributes = {
+            "string": "value",
+            "bool": True,
+            "int": 1,
+            "float": 2.3,
+            "string_array": ["a", "b", "c"],
+            "bool_array": [True, False, True],
+            "int_array": [1, 2, 3],
+            "float_array": [1.2, 2.3, 3.4],
+            "dict": {"a": "b"},
+        }
+
+        data = [
+            self._create_gauge(value=20, attributes=attributes),
```

Comment:
should we try all datapoint types (+monotonic/non-monotonic variations) to make sure all paths are covered? :thinking: